backend/lib/edgehog/containers/deployment/changes/create_deployment_on_device.ex

```diff
@@ -26,11 +26,8 @@ defmodule Edgehog.Containers.Deployment.Changes.CreateDeploymentOnDevice do
 
   @impl Ash.Resource.Change
   def change(changeset, _opts, _context) do
-    # After the transaction has been executed, i.e. all checks have passed
-    # and the deployment is in the data layer, start the deployment with
-    # the result.
     Ash.Changeset.after_action(changeset, fn _changeset, deployment ->
-      with :ok <- Containers.send_deploy_request(deployment) do
+      with :ok <- Containers.send_deploy_request(deployment, tenant: deployment.tenant_id) do
```

Comment:
```suggestion
      with :ok <- Containers.send_deploy_request(deployment) do
```

why would you need to specify the tenant? It would be a data error if it was different from the deployment's tenant in `send_deploy_request`